docs/recipe_development.rst

```diff
@@ -0,0 +1,132 @@
+Such subclasses would be a valuable resource for many researchers, and might justify development of a Python package that could be distributed to the community.
+
+
+mBuild has been developed with this in mind, in the form of a plug-in recipe system.
+Detailed below are the specifications of this system, how to convert your own Python project into an mBuild-discoverable plug-in, and an example.
+
+Entry Points
+--------
+
+The basis of the recipe system in mBuild relies on `setuptools.entry_points <https://packaging.python.org/guides/creating-and-discovering-plugins/#using-package-metadata>`_.
+This allows other packages to register themselves with the ``entry_point`` group we defined in mBuild, so they are accessible through the ``mbuild.recipes.my_recipe_foo`` location.
+In this case, ``my_recipe_foo`` would be the recipe you want to share.
+You can install many mBuild recipes, and only have to import mBuild to have access to them all!
+It is not necessary to use calls like this: ``from my_foo_project import my_recipe_foo``.
+The call ``import mbuild`` discovers all recipes that fit the ``entry_point`` group specification and makes them available under ``mbuild.recipes``. 
+
+Registering a Recipe
+________
+
+
+In the case that a user already has a Python project set up with a structure similar to the layout below:
+
+This project can be found `here <https://github.com/justinGilmer/mbuild-fcc>`_.
+
+::
+
+    mbuild_fcc
+    ├── LICENSE
+    ├── README.md
+    ├── mbuild_fcc
+    │   ├── mbuild_fcc.py
+    │   └── tests
+    │       ├── __init__.py
+    │       └── test_fcc.py
+    └── setup.py
+
+
+The two important files for the user to convert their ``mBuild`` recipe to a discoverable plug-in are ``setup.py`` and ``mbuild_fcc.py``.
+
+To begin, lets first inspect the ``mbuild_fcc.py`` file, a shortened snippet is below.
+
+::
+
+    import mbuild
+
+
+    class FCC(mbuild.Compound):
+        """Create a mBuild Compound with a repeating unit of the FCC unit cell.
+
+        ... (shortened for viewability)
+
+        """
+
+        def __init__(self, lattice_spacing=None, compound_to_add=None, x=1, y=1, z=1):
+            super(FCC, self).__init__()
+
+            # ... (shortened for viewability)
+
+    if __name__ == "__main__":
+        au_fcc_lattice = FCC(lattice_spacing=0.40782,
+                             compound_to_add=mbuild.Compound(name="Au"),
+                             x=5, y=5, z=1)
+        print(au_fcc_lattice)
+
+
+
+There are two very important lines in this file that we need to know when developing a plug-in for mBuild.
+
+The first is the import statement ``import mbuild``.
+We must make sure that mbuild is installed since we are inheriting from ``mbuild.Compound``. When you decide to distribute your plug-in,
+the dependencies must be listed.
+
+The second is the name of the plug-in itself. It is considered good practice to name it the name of your ``class``.
```

Comment:
```suggestion
The second is to select name of the plug-in itself. It is considered good practice to name it the name of your ``class``.
```
I think this makes it more clear what's going on here, since here the developer needs to decide a name; it doesn't exist prior